README.md

```diff
@@ -25,13 +21,12 @@ python setup.py install
 
 ```python
 from coffeehouse.lydia import LydiaAI
```

Comment:
The problem with providing an API Key for each module such as `LydiaAI` is that the process can become redundant as more is added. 

I rather do this
```python
from coffeehouse.api import API
from coffeehouse.lydia import LydiaAI
from coffeehouse.tts import TextToSpeech

coffeehouse_api = API("1234567890")

lydia = LydiaAI(coffeehouse_api)
tts = TextToSpeech(coffeehouse_api)
```

Instead of this
I rather do this
```python
from coffeehouse.lydia import LydiaAI
from coffeehouse.tts import TextToSpeech

lydia = LydiaAI("1234567890")
tts = TextToSpeech("1234567890")
```